presto-phoenix/src/test/java/io/prestosql/plugin/phoenix/TestPhoenixDistributedQueries.java

```diff
@@ -134,10 +134,9 @@ public void testCreateSchema()
     }
 
     @Override
-    public void testColumnName(String columnName)
+    protected boolean isColumnNameRejected(Exception exception, String columnName, boolean delimited)
     {
-        // TODO (https://github.com/prestosql/presto/issues/3466) Phoenix generally lacks quoting in underlying queries
-        throw new SkipException("TODO");
+        return columnName.equals("a\"quote");
```

Comment:
Leave some TODO comment still.
Currently the `"` in column name leads to SQL injection.
Use of `a"quote` column should produce a reasonable exception message like "Invalid column name" or "Column name cannot contain a quotation sign".  Then, here you should verify the actual `exception` message.